website/project/views/register.py

```diff
@@ -148,37 +137,22 @@ def node_register_template_page(auth, node, **kwargs):
                 )
             except NoResultsFound:
                 raise not_found_error
-    else:
-        # Anyone with view access can see this page if the current node is
-        # registered, but only admins can view the registration page if not
-        # TODO: Test me @jmcarp
-        if not node.has_permission(auth.user, ADMIN):
-            raise HTTPError(http.FORBIDDEN)
-        registered = False
-        payload = None
-        metaschema_query = MetaSchema.find(
-            Q('name', 'eq', template_name)
-        ).sort('-schema_version')
-        if metaschema_query:
-            meta_schema = metaschema_query[0]
-        else:
-            raise not_found_error
-    schema = meta_schema.schema
-
-    # TODO: Notify if some components will not be registered
-
-    ret = {
-        'template_name': template_name,
-        'schema': json.dumps(schema),
-        'metadata_version': meta_schema.metadata_version,
-        'schema_version': meta_schema.schema_version,
-        'registered': registered,
-        'payload': payload,
-        'children_ids': node.nodes._to_primary_keys(),
-    }
-    ret.update(_view_project(node, auth, primary=True))
-    return ret
 
```

Comment:
What is the use case for a non-admin user to be able to access this view? (`@must_be_contributor_or_public` is the decorator that governs access to this view)

As far as I can tell only admins can read, edit, or even see the list of ongoing draft registrations.

Aside from any redirects to dealing with URL hacking, can we restrict this view to `@must_have_permission(ADMIN)` entirely?